fhand = open('D:\Programming\Python\Github\Python_General\Scientific Computing\\tryExcept.py', 'r')
print(fhand)
fhand.close()